pubspec.yaml

```diff
@@ -274,7 +274,7 @@ dependency_overrides:
   matrix_link_text:
     git: 
       url: https://github.com/linagora/matrix_link_text.git
-      ref: feat/add-text-span-builder
+      ref: TW-795-center-links-in-web
```

Comment:
should create `twake-supported` branch and use it

Reply:
You are right